.gitignore

```diff
@@ -110,3 +110,6 @@ cabal.project.local
 # Nix output symlinks
 result
 result-*
+
+# emacs misc
+.dir-locals.el
```

Comment:
Not opposed to the change really, but note that:

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

From: https://git-scm.com/docs/gitignore.html